pages/platform/private-registry/configure-oidc-provider-authentication/guide.en-au.md

```diff
@@ -0,0 +1,288 @@
+---
+title: 'Configure the authentication via an OIDC provider on an OVHcloud Managed Private Registry'
+excerpt: 'Find out how to configure the authentication via an OIDC provider on an OVHcloud Managed Private Registry'
+updated: 2023-08-25
+---
+
+<style>
+ pre {
+     font-size: 14px;
+ }
+ pre.console {
```

Comment:
Is it used in this doc ?

Reply:
It will be used when I will add terraform way :)